### ISSUE TITLE
Chapters in the readme

### ISSUE THREAD

**Issue by janpsegers** [bixb922/crank-organ#5]
Nice to meet you here too. In my opinion there are two good reasons to read the readme one is you want to know what the software is used for the other you want to use it, for keeps or just for a short time. In the last case it should be nice to have a chapter with a description how to install and start up at the top of the read me. Perhaps with a link to the start of the manual. I am a person that likes to learn by trial and error, so install and see what you got. In the current situation i had to read a lot of stuff before I found the installation instructions. 

By the way i love the software.

**Comment by bixb922** [comment]
You are right, the documentation is a bit long and not very well ordered. I am reluctant to split it in several documents, because I like to be able to print it out and read on paper. Perhaps putting an index at the beginning will help to have better shortcuts to the part that is interesting to the reader and could give better oversight of the contents?

There is now a way to use software even without purchasing a microcontroller, see https://drehorgel.pythonanywhere.com/ (plus some other stuff).

**Comment by janpsegers** [comment]
But most of all it’s a beautiful peace of work. It’s funny to see that several people have the same trouble to get it started. Me myself couldn’t find the ap because i was too much taken by the long list of start messages on the console. Furthermore I always found it terribly to document my work when it was finished so I admire you documentation. There’s another thing i stumbled upon. Did you take any measures to prevent a second logon in the server. In fact the server is a wireless connected user interface of a machine meant to be handled by one person but in general networking can do more. I realise its pure cosmetic but ……Thanks again for all your efforts Jan P SegersOp 11 dec 2024 om 16:38 heeft bixb922 ***@***.***> het volgende geschreven:﻿
You are right, the documentation is a bit long and not very well ordered. I am reluctant to split it in several documents, because I like to be able to print it out and read on paper. Perhaps putting an index at the beginning will help to have better shortcuts to the part that is interesting to the reader and could give better oversight of the contents?
There is now a way to use software even without purchasing a microcontroller, see https://drehorgel.pythonanywhere.com/ (plus some other stuff).

**Comment by bixb922** [comment]
Yep,the console messages are a bit on the wordy side, I use those for debugging and tend to leave those, why throw the life jacket away before the real journey begins...

>. Did you take any measures to prevent a second logon in the server. 

You can connect several browsers or devices, resources permitting. I don't know the real limitation, but two or three clients simultaneously seem to he no problem. On the "system info" page tou could see the clients

**Comment by janpsegers** [comment]
I wanted to say that it is not convenient to have more than one client fiddling on machine eg one giving a performance and a second changing the mapping Jan P SegersOp 11 dec 2024 om 20:59 heeft bixb922 ***@***.***> het volgende geschreven:﻿
Yep,the console messages are a bit on the wordy side, I use those for debugging and tend to leave those, why throw the life jacket away before the real journey begins...

. Did you take any measures to prevent a second logon in the server. 

You can connect several browsers or devices, resources permitting. I don't know the real limitation, but two or three clients simultaneously seem to he no problem. On the "system info" page tou could see the clients

**Comment by bixb922** [comment]
Since changing configuration is password protected, you can control unwanted configuration changes. And both home networks and cell phone hotspots have their password, so access is restricted anyhow to persons you'd normally trust. So  I don't see a problem in this regard (so far).

**Comment by janpsegers** [comment]
What I wanted to make clear that the software is a remote user interface for one machine and one operator. I understand that you don’t care. It’s more or less a hypothetical problem so let it be. Thanks for your reactions. Jan P SegersOp 11 dec 2024 om 21:36 heeft bixb922 ***@***.***> het volgende geschreven:﻿
Since changing configuration is password protected, you can control unwanted configuration changes. And both home networks and cell phone hotspots have their password, so access is restricted anyhow to persons you'd normally trust. So  I don't see a problem in this regard (so far).

**Comment by bixb922** [comment]
Aahhh I think now I get it, you are right!
